tinker/faculty_bios/faculty_bio_controller.py

```diff
@@ -122,21 +123,121 @@ def _iterate_child_xml(self, child, author):
             school_array = []
             for school in child.findall('.//job-titles/school'):
                 school_array.append(school.text or 'Other')
+            if csv:
+                return self.csv_file(child)
+            else:
+                page_values = {
+                    'author': child.find('author') or None,
+                    'id': child.attrib['id'] or "",
+                    'title': child.find('title').text or None,
+                    'created-on': child.find('created-on').text or None,
+                    'path': 'https://www.bethel.edu' + child.find('path').text or "",
+                    'schools': school_array,
+                    'last-name': child.find('.//last').text or None,
+                    'deactivated': child.find('.//deactivate').text or None
+                }
 
-            page_values = {
-                'author': child.find('author') or None,
-                'id': child.attrib['id'] or "",
-                'title': child.find('title').text or None,
-                'created-on': child.find('created-on').text or None,
-                'path': 'https://www.bethel.edu' + child.find('path').text or "",
-                'schools': school_array,
-                'last-name': child.find('.//last').text or None,
-                'deactivated': child.find('.//deactivate').text or None
-            }
+            # Returns the page_values
             return page_values
         else:
             return None
 
+    def csv_file(self, child):
+        # Initial dictionary values
+        page_values = {
+            'first': self.get_faculty_data(child, 'first'),
+            'last': self.get_faculty_data(child, 'last'),
+            'highlight': self.get_faculty_data(child, 'highlight'),
+            'email': self.get_faculty_data(child, 'email'),
+            'started-at-bethel': self.get_faculty_data(child, 'started-at-bethel'),
+            'location': '',
```

Comment:
Is there a reason why location is empty?

Reply:
Location is initially set to be empty in case the location field doesn't have a value in it, then down below we check if there is a value/multiple values and assign those value(s).

Reply:
Oh! I missed that section. Nevermind

Reply:
Oh. I guess i missed that. My bad